modules/installation-gcp-config-yaml.adoc

```diff
@@ -119,7 +118,7 @@ endif::private[]
 ====
 If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger machine types, such as `n1-standard-8`, for your machines if you disable simultaneous multithreading.
 ====
-<5> Optional: The custom encryption key section to encrypt both virtual machines and persistent volumes.
+<5> Optional: The custom encryption key section to encrypt both virtual machines and persistent volumes. Your default compute service account must have the permissions granted to use your KMS key and have the correct IAM role assigned. The default service account name follows the `service-<project_number>@compute-system.iam.gserviceaccount.com` pattern. For more information on granting the correct permissions for your service account, see _Machine management_ -> _Creating machine sets_ -> _Creating a machine set on GCP_.
```

Comment:
Does someone know what KMS means or IAM?

Are we using italics for our fake-links? I've been using double quotes. ("Machine management" -> ...)

Reply:
Yeah, KMS and IAM are very common terms in the GCP realm. If someone is installing to GCP, they should most definitely know what they are.

Good call on the double quotes; the italics have been the most common practice I've seen for this, but the ISG recommends double quotes for chapter and section titles. Thanks for pointing that out!